staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -946,6 +946,12 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume `json:"allowedFlexVolumes,omitempty" protobuf:"bytes,18,rep,name=allowedFlexVolumes"`
+	// allowedUnsafeSysctls is a list of explicitly allowed unsafe sysctls, defaults to none, * means all
```

Comment:
document suffix globbing on "*" (e.g. "foo/*" allows "foo/bar", "foo/baz")

Reply:
also `kernel.msg*` with the slash is possible

Reply:
Added both to the examples